### ISSUE TITLE
.gitignore much bro?

### ISSUE THREAD

**Issue by ZetaHunter** [agjs/eSpider#1]
Hey, it seems to me you forgot to .gitignore node modules directory.

**Comment by agjs** [comment]
Yeah, just created the repo yesterday and will build this when I get some time in the upcoming days :). If you are willing to team up and build this, you are welcome.

**Comment by ZetaHunter** [comment]
Would love to but I am more of Flask (Py3.4) / MarionetteJS kind of guy, don't have much experience in Node/IOS/ElectronJS.

**Comment by agjs** [comment]
Alrighty sir, fair enough :). When I manage to get some time and build this, make sure to stay tuned and drop some suggestions etc.
